code/modules/projectiles/projectile.dm

```diff
@@ -124,23 +124,40 @@
 
 	var/temporary_unstoppable_movement = FALSE
 
-	///If defined, on hit we create an item of this type then call hitby() on the hit target with this
+	///If defined, on hit we create an item of this type then call hitby() on the hit target with this, mainly used for embedding items (bullets) in targets
 	var/shrapnel_type
+	///If we have a shrapnel_type defined, these embedding stats will be passed to the spawned shrapnel type, which will roll for embedding on the target
+	var/list/embedding = NONE
+
 	///If TRUE, hit mobs even if they're on the floor and not our target
 	var/hit_stunned_targets = FALSE
 
-	wound_bonus = CANT_WOUND
-	/// For telling whether we want to roll for bone breaking or lacerations if we're bothering with wounds
-	var/sharpness = FALSE
+	wound_bonus = CANT_WOUND // can't wound by default

```

Comment:
Since this is a redefined variable, put it at the top, before the new vars, for clarity.